test/Eris/Generator/SuchThatGeneratorTest.php

```diff
@@ -20,6 +20,18 @@ function($n) { return $n % 2 == 0; },
         );
     }
 
+    public function testAcceptsPHPUnitConstraints()
+    {
+        $generator = new SuchThatGenerator(
+            $this->callback(function($n) { return $n % 2 == 0; }),
+            ConstantGenerator::box(10)
+        );
+        $this->assertEquals(
```

Comment:
@giorgiosironi 

Ha, right - not sure why I wrote `'3'` instead of `'10'`. Updated my comment!